### PR TITLE
[Storybook 8] Composites/Adapters

### PR DESCRIPTION
**Pull request by JamesBurnside**
# What

- Migrate Adapters Page to Storybook 8
- Add `.txt` loader to webpack config for loading txt snippets
- Add `<Note />` component for Notes, the default message bar styles weren't good so created this wrapper component

# Why

Migration to Storybook 8

# How Tested

Checked locally:

image

### CODE REVIEW

**Review by dmceachernmsft — APPROVED** [review]

File: packages/storybook8/stories/Composites/Adapters/Docs.mdx [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+import ChatAdapterExampleText from '!!raw-loader!./snippets/ChatAdapterExample.snippet.tsx';
+import ChatAdapterStateChangesText from '!!raw-loader!./snippets/ChatAdapterStateChanges.snippet.txt';
+
+<Meta title="Composites/Adapters" />
+
+# Adapters for Composites
+
+Composites deliver turn-key communication experiences for developers.
+They make it easy for developers to add end-to-end communication experiences to their apps with only a couple lines of code.
+To power Composites, the UI library provides `Adapters`, one for each Composite. These adapters make use of the stateful clients for calling and chat in conjunction with React Hooks like usePropsFor to provide developers a single interface to configure Composites.
```
Comment (JoshuaLai) [inline packages/storybook8/stories/Composites/Adapters/Docs.mdx:15]: nitpick: The original content had
` one for the calling composite [here](./?path=/docs/composites-call-basicexample--basic-example) and one for chat composite
[here](./?path=/docs/composites-chat-basicexample--basic-example).`
Are we ok with the content change here to not include the links?

File: packages/storybook8/stories/Composites/Adapters/Docs.mdx [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+
+#### Call Adapter Examples
+
+<Source code={CallAdapterStateChangesText} />
+
+#### Chat Adapter Examples
+
+<Source code={ChatAdapterStateChangesText} />
+
+The ChatAdapter and the ChatComposite are purpose-built to support single chat thread. Only events, such as 'messagesReceived', from the thread that initialized the ChatAdapter, will be triggered. To accommodate multiple threads usage, you can leverage the foundational service assistant, the Chat [StatefulClient](./?path=/docs/statefulclient-overview--page), in conjunction with [usePropsFor](./?path=/docs/statefulclient-reacthooks-usepropsfor--page) to render the Chat UI Component for an enhanced chatting experience. Alternatively, you can manage multiple adapters corresponding to different threads, and display the Chat UI Composite based on the threads the user wishes to access.
```
Comment (JoshuaLai) [inline packages/storybook8/stories/Composites/Adapters/Docs.mdx:81]:
```suggestion
The ChatAdapter and the ChatComposite are purpose-built to support single chat thread. Only events, such as 'messagesReceived', from the thread that initialized the ChatAdapter, will be triggered. To accommodate multiple threads usage, you can leverage the foundational service assistant, the Chat [StatefulClient](./?path=/docs/stateful-client-overview--docs), in conjunction with [usePropsFor](./?path=/docs/statefulclient-reacthooks-usepropsfor--page) to render the Chat UI Component for an enhanced chatting experience. Alternatively, you can manage multiple adapters corresponding to different threads, and display the Chat UI Composite based on the threads the user wishes to access.
```

File: packages/storybook8/stories/Composites/Adapters/Docs.mdx (hunk 1 above)
Comment (JamesBurnside) [reply packages/storybook8/stories/Composites/Adapters/Docs.mdx:15]: It excluded `callwithchat` so I shortened to just `one for each composite` instead. Can revert if you like

**Review by JoshuaLai — APPROVED** [review]
Just a nit pick comment and some update to links! Looks good

**Pull request merged**